zulipterminal/helper.py

```diff
@@ -421,3 +429,21 @@ def canonicalize_color(color: str) -> str:
         return color.lower()
     else:
         raise ValueError('Unknown format for color "{}"'.format(color))
+
+
+@asynch
+def notify(title: str, html_text: str) -> None:
+    document = lxml.html.document_fromstring(html_text)
+    html_text = document.text_content()
```

Comment:
One reason to change the function parameter name, was that it would separate the input format (html_text) from the `text_content()` result, ie. just 'text'. That is the case, right?

Reply:
Oh! Now, I understand what you meant by the renames. 😬